components/common-styles/_social-icons.scss

```diff
@@ -0,0 +1,11 @@
+%social-icon--facebook {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' viewBox='0 0 266.89 266.89'%3E%3Cdefs%3E%3Cstyle%3E.cls-1%7Bfill:none;%7D.cls-2%7Bclip-path:url(%23clip-path);%7D.cls-3%7Bfill:%23fff;%7D%3C/style%3E%3CclipPath id='clip-path' transform='translate(0 0)'%3E%3Crect class='cls-1' width='266.89' height='266.89'/%3E%3C/clipPath%3E%3C/defs%3E%3Ctitle%3Etsc-fb-icon%3C/title%3E%3Cg id='Layer_2' data-name='Layer 2'%3E%3Cg id='Layer_1-2' data-name='Layer 1'%3E%3Cg class='cls-2'%3E%3Cpath class='cls-3' d='M252.16,0H14.73A14.73,14.73,0,0,0,0,14.73V252.16A14.73,14.73,0,0,0,14.73,266.9H142.56V163.54H107.78V123.26h34.78V93.55c0-34.47,21.05-53.24,51.81-53.24a285.5,285.5,0,0,1,31.08,1.59v36H204.12c-16.72,0-20,7.95-20,19.61v25.72H224l-5.19,40.28H184.15V266.9h68a14.73,14.73,0,0,0,14.73-14.73V14.73A14.73,14.73,0,0,0,252.16,0' transform='translate(0 0)'/%3E%3C/g%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
+}
```

Comment:
You can save characters by removing the title tag and also the bit with `id='Layer_2' data-name='Layer 2'`